integers.md

```diff
@@ -148,7 +148,7 @@ $ go test -v
 
 Notice the special format of the comment, `// Output: 6`. While the example will always be compiled, adding this comment means the example will also be executed. Go ahead and temporarily remove the comment `// Output: 6`, then run `go test`, and you will see `ExampleAdd` is no longer executed.
 
-To view example documentation, let's take a quick look at `godoc`. Run `godoc -http=:6060` and open a web browser to `http://localhost:6060/pkg/`. Inside here you'll see a list of all of Go's Standard Library packages, plus Third Party packages you have installed, under which you should see your example documentation for `learn-go-with-tests`. Now look under `Integers`, then under `func Add`, then expand `Example` and you should see the example you added for `sum := Add(1, 5)`.
+To view example documentation, let's take a quick look at `pkgsite`. Navigate to your project's directory, then run `pkgsite -open .`, which should open a web browser for you, pointing to `http://localhost:8080`. Inside here you'll see a list of all of Go's Standard Library packages, plus Third Party packages you have installed, under which you should see your example documentation for `github.com/quii/learn-go-with-tests`. Follow that link, and then look under `Integers`, then under `func Add`, then expand `Example` and you should see the example you added for `sum := Add(1, 5)`.
```

Comment:
In my case, opening the `pkgsite` documentation does not show any of the standard libraries. Does it do that for you?

Reply:
Yes, it does; pkgsite showing the standard library is the bare minimum of what it can do locally.

Reply:
Hm, for me, in what I believe to be a standard setup, it just shows the package I'm currently at, with no standard library included. 

Anything specific we could compare between our setups here? I don't mean to troubleshoot my potential issues, rather replicate a situation in which a user with similar setup to mine would find the stdlib packages missing